sql_generators/glean_usage/templates/events_first_seen_v1.query.sql

```diff
@@ -0,0 +1,130 @@
+{{ header }} -- the header doesn't seem to work properly
+
+{% raw %}
+{% if is_init() %}
+{% endraw %}
+with
+eventsstream as (
+    select
+    client_id,
+    profile_group_id,
+    sample_id,
+    event_category,
+    event_name,
+    `event`,
+    cast(null as string) as criteria,
+    min(submission_timestamp) as first_submission_timestamp,
+    min(event_timestamp) as first_event_timestamp,
+    min_by(event_extra, submission_timestamp) as event_extra,
+    min_by(app_version_major, submission_timestamp) as app_version_major,
+    min_by(normalized_channel, submission_timestamp) as normalized_channel,
+    min_by(normalized_country_code, submission_timestamp) as normalized_country_code,
+    min_by(normalized_os, submission_timestamp) as normalized_os,
+    min_by(normalized_os_version, submission_timestamp) as normalized_os_version,
+    from
+    `{{ project_id }}.{{ app_name }}_derived.{{ base_table }}`
+    where
+    -- initialize by looking over all of history
+    date(submission_timestamp) >= date_sub(@submission_date, interval 1 day)
```

Comment:
I think this will fail deployment because is_init doesn't add the submission_date parameter.

Also is this meant to be `<=`  based on the comment or do you only want the most recent day to be initialized?